src/core/include/openvino/core/graph_util.hpp

```diff
@@ -304,4 +304,9 @@ OPENVINO_API
 void save_model(const std::shared_ptr<const ov::Model>& model,
                 const std::string& output_model,
                 bool compress_to_fp16 = true);
+
```

Comment:
looks like we also need to guard with (see https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/core.hpp#L67-L83)
```c++
#if defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
```

Reply:
Done.